### PR TITLE
This installs the Chrome browser extension shim on Windows

### PR DESCRIPTION
**Pull request by zanderz**
Depends on the Windows path PR before we ship

### CODE REVIEW

File: go/kbnm/main.go [hunk 1]
```diff
@@ -86,6 +86,11 @@ func main() {
 		out = nativemessaging.NewNativeJSONEncoder(os.Stdout)
 	}
 
+	if *versionFlag {
```
Comment (shazow) [inline go/kbnm/main.go]: This is going to conflict with https://github.com/keybase/client/pull/6524 which was merged, FYI. Ideally keep my version, or use Println/add a newline.

File: go/kbnm/main.go [hunk 2]
```diff
@@ -11,7 +11,7 @@ import (
 )
 
 // Version is the build version of kbnm, overwritten during build.
-const Version = "dev"
+var Version = "dev"
```
Comment (shazow) [inline go/kbnm/main.go]: Ugh oops, good catch.

File: go/kbnm/windows/kbnm.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (shazow) [inline go/kbnm/windows/kbnm.json]: Any way to generate this during build from the template? It's already out of date. :/

If not, need to update it to https://github.com/keybase/client/blob/19c972b13c082b837346996e5af7c46fae5ebae8/go/kbnm/host_json.template which includes the upcoming prod extension id.

Comment (shazow) [inline go/kbnm/windows/kbnm.json]: Another option: We could put it ingo/kbnm/host_json.windows (or something right next to the original template) and then copy it over during the `build_prerelease.cmd` phase. At least this way it'll be all in one place-ish and we'll remember to keep them in sync.

File: go/kbnm/main.go (hunk 1 above)
Comment (zanderz) [inline go/kbnm/main.go]: ok I'll take that back out

File: go/kbnm/windows/kbnm.json (hunk 3 above)
Comment (zanderz) [inline go/kbnm/windows/kbnm.json]: ok, good idea

**Pull request merged**